doc/en/components/layouts/tile-manager.md

```diff
@@ -0,0 +1,335 @@
+---
+title: {Platform} Tile Manager Component - {ProductName}
+_description: {Platform} Tile Manager component enables the display of content in individual tiles.
+_keywords: {Platform} Tile Manager, {ProductName}, Infragistics
+mentionedTypes: ["TileManager"]
+---
+
+# {Platform} Tile Manager Overview
+
+The {Platform} Tile Manager component enables the display of content in individual tiles. It allows users to interact with these tiles by rearranging and resizing them, giving them the freedom to customize the layout and appearance of the content according to their preferences. This flexibility enhances the user experience by enabling a more personalized and efficient way to view and manage content.
+
+## {Platform} Tile Manager Example
+
+The following {ProductName} Tile Manager Example shows the component in action. 
+
+`sample="/layouts/tile-manager/overview", height="743", alt="{Platform} Tile Manager Example"`
```

Comment:
We need to add scrolling capabilities here, as once tiles are resized higher the bottom part of the tile manager remains out of view.
![image](https://github.com/user-attachments/assets/626b81ab-4758-473c-a1f5-8710f6e28afe)